From 5188111cfc4a3ae15b4f2a2f4c881753dc144b8e Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Thu, 21 Dec 2023 16:21:33 +0100
Subject: [PATCH] com.webos.service.mediacontroller.role.json.in: Fix outbound
 permission issues

Fixes:

Dec 21 14:59:53 qemux86-64 ls-hubd[242]: [] [pmlog] ls-hubd LSHUB_NO_OUT_PERMS {"DEST_APP_ID":"com.webos.monitor1123","SRC_APP_ID":"com.webos.service.mediacontroller","EXE":"/usr/sbin/com.webos.service.mediacontroller","PID":1091} "com.webos.service.mediacontroller" does not have sufficient outbound permissions to communicate with "com.webos.monitor1123" (cmdline: /usr/sbin/com.webos.service.mediacontroller)

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Pending

 files/sysbus/com.webos.service.mediacontroller.role.json.in | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/files/sysbus/com.webos.service.mediacontroller.role.json.in b/files/sysbus/com.webos.service.mediacontroller.role.json.in
index e15e1b8..79b8ed1 100644
--- a/files/sysbus/com.webos.service.mediacontroller.role.json.in
+++ b/files/sysbus/com.webos.service.mediacontroller.role.json.in
@@ -7,7 +7,9 @@
         {
             "service": "com.webos.service.mediacontroller",
             "outbound": ["com.webos.service.bluetooth2",
-                         "com.webos.service.account"]
+                         "com.webos.service.account",
+                         "com.webos.monitor*",
+                         "org.webosports.app.testr"]
         }
     ]
 }
